packages/storybook/src/primitives/Label/Label.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import { Box, Label, Flex, Input } from '@kodiak/primitives'
+
+export default { title: 'Forms/Label' }
+
+export function initial() {
+  return (
+    <Box p={4}>
+      <Label>Renders a `label` tag by default</Label>
+    </Box>
+  )
+}
+
+export function verticalLabel() {
+  return (
+    <Flex flexDirection="column" p={4}>
+      <Label mb={2} htmlFor="input">
```

Comment:
@alexanderchan should we make a default variant for a label with bottom margin and an `inline` variant for removing the bottom margin and adding the right margin?